cmake/modules/AddSwift.cmake

```diff
@@ -947,7 +947,7 @@ function(add_swift_host_tool executable)
     add_dependencies(${ASHT_SWIFT_COMPONENT} ${executable})
     swift_install_in_component(TARGETS ${executable}
                                RUNTIME
-                                 DESTINATION bin
+                                 DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
Thanks for doing this: should this be [the new `SWIFT_TOOLS_INSTALL_DIR` from the recent rebranch](https://github.com/apple/swift/pull/61769/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR325) instead?

Reply:
It's a hard to know since I don't see `SWIFT_TOOLS_INSTALL_DIR` being "Used by add_swift_tool_symlink in AddSwift.cmake": https://github.com/apple/swift/blob/3616bf1a34c16baa01feaea10b3e256f1f879cf4/cmake/modules/AddSwift.cmake#L978

And it doesn't exist on this branch. How do we handle this? 🤔

Reply:
> It's a hard to know since I don't see SWIFT_TOOLS_INSTALL_DIR being "Used by add_swift_tool_symlink in AddSwift.cmake":

`add_swift_tool_symlink` calls `llvm_add_tool_symlink` with `SWIFT` (https://github.com/apple/swift/blob/main/cmake/modules/AddSwift.cmake#L979). That then calls `llvm_install_symlink` which uses it via `set(output_dir "${${project}_TOOLS_INSTALL_DIR}")`.